precommit~

```diff
@@ -0,0 +1,7 @@
+# Auto format.
```

Comment:
I don't know exactly what this file is, but I don't think it should exist. Please remove!